src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/SentReferralSummariesDTO.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ReferralSummary
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class SentReferralSummariesDTO(
+  val id: UUID,
+  val sentAt: OffsetDateTime,
+  val sentBy: AuthUserDTO,
+  val referenceNumber: String,
+  val assignedTo: AuthUserDTO?,
+  val serviceUser: ServiceUserDTO?,
```

Comment:
nit: i think this should be non-nullable

Reply:
I was thinking about unassigned cases and thought it will not have serviceUser or serviceProvider?

Reply:
service user is the person on probation this referral was made for. so every referral has a service user associated with it. thanks for making the change.